crates/nu-command/src/filters/append.rs

```diff
@@ -32,7 +32,22 @@ only unwrap the outer list, and leave the variable's contents untouched."#
     }
 
     fn search_terms(&self) -> Vec<&str> {
-        vec!["add", "concatenate"]
+        vec![
+            "add",
+            "concatenate",
+            "push",
+            "push_back",
+            "emplace_back",
+            "list.append",
+            "set.add",
+            "conj",
+            "into",
+            "concat",
+            "concatMap",
+            "++",
+            "merge",
+            "combine",
+        ]
```

Comment:
A few additional suggestions for the commands
```suggestion
        vec![
            "add",
            "concatenate",
            "push",
            "push_back",
            "emplace_back",
            "list.append",
            "set.add",
            "conj",
            "into",
            "concat",
            "concatMap",
            "++",
            "merge",
            "combine",  
            "Append",
            "-Append",
            "insert",
            "prepend",
            "put",
            "emplace",
        ]
```

Reply:
Thanks for the review and thoughtful feedback. You're absolutely right about the case sensitivity issue. I've also incorporated your suggestions and will push the changes.